README.md

```diff
@@ -13,9 +13,10 @@ Find the greatest product of five consecutive digits in the 1000-digit number.
 - Define a Class in `largestProduct.js` named `LargestProduct` that accepts a single argument, the 1000-digit number. This number should be set as an instance property.
 
 - This Class will have two instance methods:
-	- `getLargestProductPosition`returns the position of the largest product of 5 consecutive numbers found in the 1000-digit number.  
-	- `getLargestProduct` returns the value of the largest product of 5 consecutive numbers found in the 1000-digit number.
+  - `getLargestProductIndex`returns the index of the largest product of 5 consecutive numbers in the 1000-digit number.
```

Comment:
Lets changes this to the `starting index` rather than just index so it's crystal clear what index we're asking for.